frontend/src/components/cars-categories/cars-categories.tsx

```diff
@@ -2,21 +2,40 @@ import React, { FC, useState } from 'react';
 import { Link } from 'react-router-dom';
 
 import { mockBrands } from '@components/cars-categories/mock-brands';
+import { Mousewheel, Navigation, SwiperOptions } from 'swiper';
+import { Swiper, SwiperSlide } from 'swiper/react';
 
 import { Brand } from '../../types/brand.type';
+import 'swiper/css';
+import 'swiper/css/navigation';
+import 'swiper/css/mousewheel';
```

Comment:
It's recommended to put the 3rd party imports before the project-related imports. So I would suggest putting these styles right after the `import { Swiper, SwiperSlide } from 'swiper/react';`

```suggestion
import { Swiper, SwiperSlide } from 'swiper/react';
import 'swiper/css';
import 'swiper/css/navigation';
import 'swiper/css/mousewheel';

import { Brand } from '../../types/brand.type';
import styles from './styles.module.scss';
```